lib/tasks/taxonomy.rake

```diff
@@ -30,4 +36,21 @@ namespace :taxonomy do
     Rake::Task["taxonomy:create_taxons"].invoke
     Rake::Task["taxonomy:link_taxons"].invoke
   end
+
+  def parse_base_paths_string(base_paths)
+    pair_of_paths = base_paths.split('|')
+
+    unless pair_of_paths.all? { |pair| (pair.split(',').size % 2).zero? }
+      raise ArgumentError, base_paths_error_message
+    end
+
+    pair_of_paths.map do |pair|
+      split_pair = pair.split(',')
```

Comment:
sorry @dougdroper, we missed your comment, but just wanted to let you know that we do agree with your suggestion, thank you.